examples/Google.Cloud.Functions.Examples.IntegrationTests/SimpleDependencyInjectionTest.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright 2020, Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.Functions.Invoker.Testing;
+using Microsoft.Extensions.Logging;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Google.Cloud.Functions.Examples.IntegrationTests
+{
+    public class SimpleDependencyInjectionTest
+    {
+        [Fact]
+        public async Task LogEntryIsRecorded()
+        {
+            using (var server = new FunctionTestServer<SimpleDependencyInjection.Function>())
+            {
+                // We shouldn't have any log entries at the start of the test.
```

Comment:
Maybe the functions framework itself could, at some point, be configurable to log startup?

Reply:
It does log startup, but that won't log anything into the function's category. Will clarify this comment.